server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetFieldMappingAction.java

```diff
@@ -63,6 +70,12 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         GetFieldMappingsRequest getMappingsRequest = new GetFieldMappingsRequest();
         getMappingsRequest.indices(indices).fields(fields).includeDefaults(request.paramAsBoolean("include_defaults", false));
         getMappingsRequest.indicesOptions(IndicesOptions.fromRequest(request, getMappingsRequest.indicesOptions()));
+
+        if (request.hasParam("local")) {
+            deprecationLogger.deprecatedAndMaybeLog("get_field_mapping_local",
+                "Use [local] in get field mapping requests is deprecated."
+                    + "The parameter will be removed in the next major version");
```

Comment:
Maybe add a space between the two sentences?

Reply:
Yep, added.